test/shape.test.js

```diff
@@ -5,10 +5,9 @@ const { readFileSync } = require('fs');
 const path = require('path');
 const { DOMParser } = require('@xmldom/xmldom');
 const SVGSpriter = require('../lib/svg-sprite.js');
+const { isObject } = require('../lib/svg-sprite/utils/index.js');
 const fixturesPath = require('./helpers/fixtures-path.js');
 
-const isObject = obj => typeof obj === 'object' && !Array.isArray(obj) && obj !== null;
```

Comment:
Makes me wonder if we should go with this solution?

There's also the `isPlainObject` we need to replace later, and I'm not sure of the difference between these 2 TBH.